dist/api_data.py

```diff
@@ -592,14 +592,29 @@ def __ge__(self, other):
         perform eviction in worker threads when the cache contains at least
         this much content. It is a percentage of the cache size if the value is
         within the range of 10 to 100 or an absolute size when greater than 100.
-        The value is not allowed to exceed the \c cache_size.''',
+        The value is not allowed to exceed the \c cache_size''',
         min=10, max='10TB'),
     Config('eviction_trigger', '95', r'''
         trigger application threads to perform eviction when the cache contains
         at least this much content. It is a percentage of the cache size if the
         value is within the range of 10 to 100 or an absolute size when greater
-        than 100.  The value is not allowed to exceed the \c cache_size.''',
+        than 100.  The value is not allowed to exceed the \c cache_size''',
         min=10, max='10TB'),
+    Config('eviction_updates_target', '0', r'''
+        perform eviction in worker threads when the cache contains at least
+        this many bytes of updates. It is a percentage of the cache size if the
+        value is within the range of 0 to 100 or an absolute size when greater
+        than 100. Calculated as 20% of \c eviction_dirty_target by default.
+        The value is not allowed to exceed the \c cache_size''',
+        min=0, max='10TB'),
+    Config('eviction_updates_trigger', '0', r'''
+        trigger application threads to perform eviction when the cache contains
+        at least this many bytes of updates. It is a percentage of the cache size
+        if the value is within the range of 1 to 100 or an absolute size when
+        greater than 100. Calculated as 20% of \c eviction_dirty_trigger by default.
+        The value is not allowed to exceed the \c cache_size. This setting only
+        alters behavior if it is lower than eviction_trigger''',
+        min=0, max='10TB'),
```

Comment:
These default values (20% of the dirty equivalents) were deliberately aggressive to shake out testing issues and try to measure the maximum impact.  We should discuss what defaults make sense.

Reply:
My initial reaction was that I thought it would be bigger than the dirty limit - so I guess I was off by nearly an order of magnitude :)
Do you have any suggestion about how we might go about choosing numbers here? Should we start liberal, and tune it down if we see issues in the field? It's going to be hard to go the other way.